Esterni/PianoProgetto/components/Introduzione.tex

```diff
@@ -0,0 +1,39 @@
+\section{Introduzione}
+\subsection{Scopo del documento}
+Lo scopo del documento è quello di specificare la pianificazione che \Gruppo  adotterà per sviluppare il progetto \NomeProgetto. All'interno sono presenti le strategie per la gestione dei rischi, le suddivisioni dei compiti con il relativo preventivo dell'utilizzo delle risorse, e il modello di sviluppo secondo il quale il team ha intenzione di lavorare.
+In particolare il documento tratta i seguenti argomenti:
+\begin{itemize}
+\item analisi dei rischi;
+\item descrizione del modello di sviluppo;
+\item pianificazione delle attività e divisione dei ruoli;
+\item stima dei costi e delle risorse necessarie allo sviluppo del progetto.
+\end{itemize}
+\subsection{Scopo del prodotto}
+L'obbiettivo del progetto è quello di sviluppare un algoritmo in grado di garantire la sincronizzazione e il salvataggio di file tra server e client, appoggiandosi sul servizio \textit{Zextras Drive}.
+L'interfaccia deve essere multi piattaforma, per poter essere utilizzato nei più importanti sistemi operativi desktop esistenti.
+Inoltre il prodotto deve essere sviluppato aderendo al pattern MVC, così da permettere rapidi cambiamenti della Business logic o dell'interfaccia utenti senza eccessivo sviluppo aggiunto.
```

Comment:
Non correggere per ora, dobbiamo capire se le parole in inglesi vanno scritte con la maiuscola oppure no. è da discuterne